flag_int64_slice.go

```diff
@@ -135,7 +141,52 @@ func lookupInt64Slice(name string, set *flag.FlagSet) []int64 {
 		if err != nil {
 			return nil
 		}
+		// extract default value from the flag
+		var defaultVal []int64
+		for _, v := range strings.Split(f.DefValue, ",") {
+			iV, _ := strconv.ParseInt(v, 10, 64)
+			defaultVal = append(defaultVal, iV)
+		}
+		// if the current value is not equal to the default value
+		// remove the default values from the flag
+		if !isInt64SliceEqual(parsed, defaultVal) {
+			for _, v := range defaultVal {
+				parsed = removeFromInt64Slice(parsed, v)
+			}
+		}
 		return parsed
 	}
 	return nil
 }
+
+func removeFromInt64Slice(slice []int64, val int64) (newVal []int64) {
+	var count int
+	for _, v := range slice {
```

Comment:
We should be able to drop the `count` variable by using the slice index directly:

```go
for i, v := range slice {
```

Reply:
Fixed